src/AppBundle/Subscriber/BugLabelNewIssueSubscriber.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace AppBundle\Subscriber;
+
+use AppBundle\Event\GitHubEvent;
+use AppBundle\GitHubEvents;
+use AppBundle\Issues\Status;
+use AppBundle\Issues\StatusApi;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class BugLabelNewIssueSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @var StatusApi
+     */
+    private $statusApi;
+
+    public function __construct(StatusApi $statusApi)
+    {
+        $this->statusApi = $statusApi;
+    }
+
+    /**
+     * Changes "Bug" issues to "Needs Review".
+     *
+     * @param GitHubEvent $event
+     */
+    public function onIssues(GitHubEvent $event)
+    {
+        $data = $event->getData();
+        $repository = $event->getRepository();
+        if ('labeled' !== $action = $data['action']) {
+            $event->setResponseData(array('unsupported_action' => $action));
```

Comment:
Yea, we need to work on this part a bit more - different listeners will replace each other's response data keys. Mostly, I just want a simple summary of what happened in the response, so we can use that in the functional tests. I'm not going to change this now, but yea, it's shaky.